astropy/io/fits/tests/test_image.py

```diff
@@ -1125,24 +1125,34 @@ def test_comp_image(self, data, compression_type, quantize_level,
             assert fd[1].header['BITPIX'] == chdu.header['BITPIX']
 
     @pytest.mark.skipif('not HAS_SCIPY')
+    @pytest.mark.remote_data
     def test_comp_image_quantize_level(self):
         """
         Regression test for https://github.com/astropy/astropy/issues/5969
 
         Test that quantize_level is used.
 
         """
-        import scipy.misc
+        import scipy
+        from astropy.utils import minversion
+
+        SCIPY_LT_1_10 = not minversion(scipy, '1.10dev0')
         np.random.seed(42)
-        data = scipy.misc.ascent() + np.random.randn(512, 512)*10
+
+        if SCIPY_LT_1_10:
+            import scipy.misc
+            scipy_data = scipy.misc.ascent()
+        else:
+            import scipy.datasets
+            scipy_data = scipy.datasets.ascent()
```

Comment:
```suggestion
        if SCIPY_LT_1_10:
            scipy_data = scipy.misc.ascent()
        else:
            scipy_data = scipy.datasets.ascent()
```
Seems both submodules are already imported by default.

Reply:
See https://github.com/astropy/astropy/pull/13657